frontend/src/setupTests.js

```diff
@@ -2,4 +2,4 @@
 // allows you to do things like:
```

Comment:
not sure what this file is

Reply:
It was a file I added as a suggestion from a  stackoverflow thread to get react-testing to work